src/Fantomas.Tests/FormattingPropertyTests.fs

```diff
@@ -450,7 +450,8 @@ let rec shrinkSynExpr = function
     | SynExpr.MatchLambda(_, _, clauses, _, _) -> 
         seq { yield! Seq.collect collectSynMatchClause clauses }
     | SynExpr.TryWith(expr, _, clauses, _, _, _, _)
-    | SynExpr.Match(_, expr, clauses, _, _) -> 
+    | SynExpr.Match(_, expr, clauses, _, _) 
+    | SynExpr.MatchBang(_, expr, clauses, _, _) -> 
```

Comment:
I also fix incomplete pattern match by adding a branch for `match!`.

Reply:
Perhaps add this in another PR? Added the pattern match does not mean that `match!` is supported right?